client/analytics/report/coupons/config.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { stringifyQuery } from 'lib/nav-utils';
+import { NAMESPACE } from 'store/constants';
+
+export const getCouponLabelsById = queryString => {
+	const idList = queryString
+		.split( ',' )
+		.map( id => parseInt( id, 10 ) )
+		.filter( Boolean );
```

Comment:
I really appreciate your use of filter( Boolean ) here to avoid falsy things showing up in the idList, e.g. in the event someone messes with the queryString.  Would it be a good idea to remove duplicates too?

Reply:
(It is noteworthy that even with a request containing duplicates, the REST API only returns one blob per unique ID)